### ISSUE TITLE
Feature request: Add option to output sound to stdout

### ISSUE THREAD

**Issue by BrianAllred** [jmaliksi/jlohn-mladden#6]
This would be super helpful for piping the audio into other applications.

**Comment by jmaliksi** [comment]
i honestly have no idea how to do this! can you point me in the right direction (or open up a PR)?

**Comment by jmaliksi** [comment]
unless you meant to output the words that are being said to stdout? that I can do, i just need to rearrange the debug output to actually go to a debug thing

**Comment by BrianAllred** [comment]
Either would be helpful, but the audio itself would be most helpful. But the words would be a good start, I think. I may take a stab at it, but my python is rudimentary at best, haha.

**Comment by jmaliksi** [comment]
yeah so getting the audio output has been an adventure because on my machine, it basically all has to hook into objective C native stuff, so I have no idea about any of that. I'll look into getting the words working tomorrow!

**Comment by jmaliksi** [comment]
ok i've separated vocalization text and debug stuff to stdout/stderr as appropriate. lmk if you have any resources about piping audio to stdout and i can take a look after work!
